content/blog/k8s-operator/index.md

```diff
@@ -0,0 +1,98 @@
+
+### 🟢 Simplification for the end user
+
+Abstracting your application into a single CRD helps users view your application as a single component rather than individual, separate parts (deployments/statefulsets/configmaps/etc). The operator can surface an overall system state, which reduces the cognitive load on users.
+
+In Pixie’s case, users previously checked Pixie’s deploy status by viewing the pods:
+
+```bash
+kubectl get pods -n pl
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+kelvin-5b7c8c4c5b-n7v5x                   1/1     Running   0          2d20h
+pl-etcd-0                                 1/1     Running   0          2d20h
+pl-etcd-1                                 1/1     Running   0          2d20h
+pl-etcd-2                                 1/1     Running   0          2d20h
+pl-nats-0                                 1/1     Running   0          2d20h
+vizier-certmgr-7bcbf9d4bd-r8h5s           1/1     Running   0          2d20h
+vizier-cloud-connector-854f8bb487-d69kk   1/1     Running   0          2d20h
+vizier-metadata-79f8764589-hmz59          1/1     Running   0          2d20h
+vizier-pem-crg62                          1/1     Running   12         2d20h
+vizier-pem-r2xsn                          1/1     Running   4          2d20h
+vizier-proxy-f584dc9c8-4gb72              1/1     Running   0          2d20h
+vizier-query-broker-ddbc89b-wftbz         1/1     Running   0          2d20h
+```
+
+After the addition of the CRD, the entire state of the application can be summarized with one `kubectl
+describe vizier` command:
+
+```bash
+kubectl describe vizier
+
+Status:
+ Last Reconciliation Phase Time:  2021-11-05T22:30:56Z
+ Reconciliation Phase:            Ready
+ Version:                         0.9.11
+ Vizier Phase:                    Healthy
+```
+
+### 🟢 Cleanliness and consistency
+
+Configuration options live in one place (the CRD) rather than spread out across many configmaps. The values in the CRD can be viewed as the source of truth, and is the single place where users need to make their modifications when adjusting the config.
+
+Pixie originally had four configmaps:
+
+```bash
+pl-cloud-config
+pl-cloud-connector-tls-config
+pl-cluster-config
+pl-tls-config
+```
+
+These configMaps are now represented by a [single CRD](https://github.com/pixie-io/pixie/blob/main/k8s/operator/crd/base/px.dev_viziers.yaml).
+
+### 🟢 Auto-recovery
+
+The operator can monitor the overall state of the application and apply whatever changes necessary to get the application into a healthy state. This is especially beneficial for persistent systems, or applications that need to be highly available.
+
+NATS is a major dependency of Pixie, and enables most of Pixie’s pod-to-pod communication. Occasionally we have seen the NATS instance fail, and require a redeploy to recover. The operator can monitor NATS’s status and redeploy when necessary.
```

Comment:
Reread this, just to make it even more clear let's have the last sentence be:
"The operator can monitor NATS’s status and redeploy when necessary without any action from the user".